src/components/cssComponents/Form.js

```diff
@@ -0,0 +1,79 @@
+import React, { useState } from "react";
+import App from "./../../App";
+
+function Form() {
+  
+  const [input, setInput] = useState({
+    username: "",
+    password: "",
+    repassword: "",
+  });
+ 
+  const handleChange = (e) => {
+    const { name, value } = e.target;
+    setInput({ ...input, [name]: value });
+  };
+  const submitBtn = (e) => {
+    e.preventDefault();
+  };
+  return (
+    <App header="Form" description="A simple form with validation included.">
+      <div className="content-center mb-4">
+        <form className="form-validation">
+          <div className="validation-message"></div>
+          <div className="row">
+            <label htmlFor="text">UserName</label>
+            <input
+              type="text"
+              name="username"
+              value={input.username}
+              onChange={handleChange}
+              className="form-username input-field"
+            />
+            <br />
+          </div>
+          <div className="row">
+            <label htmlFor="text">Password</label>
+            <input
+              type="password"
+              name="password"
+              value={input.password}
+              onChange={handleChange}
+              className="form-password input-field"
+              
+            />
+            <br />
+          </div>
+          <div className="row">
+            <label htmlFor="text">Re-enter Password</label>
+            <input
+              type="password"
+              name="repassword"
+              value={input.repassword}
+              onChange={handleChange}
+              className="form-rePassword input-field"
+            />
+            <br />
+          </div>
+          <button type="submit" className="form-btn " onClick={submitBtn}>
+            Sumbit

```

Comment:
You can use a Primary button for Submit button in the form. Also there is a typo it should be Submit.

Reply:
thanks, I will amend it.